gobblin-runtime/src/main/java/gobblin/runtime/api/StandardRunControls.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.runtime.api;
+
+import java.util.concurrent.TimeoutException;
+
+import gobblin.annotation.Alpha;
+
+/** Defines a standard set of methods for running Gobblin components. */
+@Alpha
+public interface StandardRunControls {
```

Comment:
Seems similar to Guava's `Service` or even `AbstractService`, can those be used instead? They are already extensively used throughout Gobblin.